easybuild/easyconfigs/l/libbaseencode/libbaseencode-1.0.11-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,35 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'libbaseencode'
+version = '1.0.11'
+
+homepage = 'https://github.com/paolostivanin/libbaseencode'
+description = """Library written in C for encoding and decoding 
+data using base32 or base64 according to RFC-4648"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/paolostivanin/libbaseencode/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['645329236992ee85f05569475da2af0f0d81aaccd4aebc8d0b8603e54a320935']  # v1.0.11.tar.gz
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.35'),
+]
+
+parallel = 1
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib/libbaseencode.%s' % SHLIB_EXT],
+    'dirs': ['lib'],
```

Comment:
I don't think there is a need to check for lib since you already check for a file in it. You may want to check for include (or a specific header though)

Reply:
See below.